packages/devtools_app/lib/src/screens/memory/panes/diff/widgets/snapshot_control_pane.dart

```diff
@@ -87,6 +87,9 @@ class SnapshotControlPane extends StatelessWidget {
   }
 }
 
+@visibleForTesting
+const classFilterButtonKey = Key('classFilterButton');
```

Comment:
From what I recall, we typically try and avoid using `Key`s just for testing (cc @kenzieschmoll)

Reply:
replaced with type